### PR TITLE
Update README.md delete changlog and release planning

### PR DESCRIPTION
**Pull request by juanpelegrina**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -10,4 +10,4 @@ The status of the solutions to all the issues currently open in the helpdesk can
...
 
-See the [changelog](changelog.md) and the [release planning](release_planning.md) for the current features and the future development/release plans for the ATS & ETS development corresponding to the [Staging instance](http://staging-inspire-validator.eu-west-1.elasticbeanstalk.com/etf-webapp/) and the [Production instance](http://inspire.ec.europa.eu/validator) of the INSPIRE Reference Validator. For the roadmap of the ETF software, refer to the [project board](https://github.com/orgs/etf-validator/projects/2) of the ETF-Validator project.
+Please check [Releases](https://github.com/inspire-eu-validation/community/releases) for current features and the [Release Strategy](https://github.com/inspire-eu-validation/community/tree/master/release%20strategy) for future deployments of the INSPIRE Reference Validator. For the roadmap of the ETF software, refer to the [project board](https://github.com/orgs/etf-validator/projects/2) of the ETF-Validator project.
```
Comment (MarcoMinghini) [inline README.md]: Please check the [changelog of the current and past releases](https://github.com/inspire-eu-validation/community/releases) and the [release planning strategy](https://github.com/inspire-eu-validation/community/tree/master/release%20strategy) for the plan of future releases of the INSPIRE Reference Validator.

**Review by MarcoMinghini — CHANGES_REQUESTED** [review]
I would rephrase the last sentence as suggested.

**Review by jenriquesoriano — APPROVED** [review]
It's ok for me, I agree with Marco's proposal

**Comment by juanpelegrina** [comment]
Thanks, I will implement the suggested  changes first thing in the morning

**Review by MarcoMinghini — APPROVED** [review]
